charts/nfs-subdir-external-provisioner/README.md

```diff
@@ -19,6 +19,7 @@ This charts installs custom [storage class](https://kubernetes.io/docs/concepts/
 
 - Kubernetes >=1.9
 - Existing NFS Share
+- nfs-common (Debian based) or nfs-utils (RedHat based) installed on all nodes
```

Comment:
I'm not sure I'm happy with that wording. Whether it's due to the lack of uniformity in capital letters or I'm wondering if it's more correct to write Red Hat with a space in the middle or at all it should be referred to more as "rpm based". Can you try to suggest a different wording?